modules/docs/mdx/10.0-UPGRADE_GUIDE.mdx

```diff
@@ -70,6 +71,16 @@ Preview and the `Menu` in Main.
 
 ## Deprecations
 
+### StatusIndicator (Main)
+
+We've removed the `StatusIndicator` component in Main. We recommend using the
+[StatusIndicator](/components/status-indicator/) in the Preview package instead; it is a
+[compound component](/getting-started/for-developers/resources/compound-components/) and offers a
+more flexible API.
+
+This change is **not** handled by the codemod due to the differences in API between the
+`StatusIndicator` in Main and the `StatusIndicator` in Preview.
```

Comment:
if we're deprecating, i think we can write a codemod since it's going from a config based component to a compound component but not sure if this is a separate ticket

Reply:
Sounds good. I'll create an issue